<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>965</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="selectFolderButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select image folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>20</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>251</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>781</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="channelsGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Channels</string>
    </property>
    <widget class="QCheckBox" name="RedChannelBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Red</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="GreenChannelBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Green</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="BlueChannelBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Blue</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="AnalysisWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>361</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="DefaultAnalysisWidget"/>
    <widget class="QWidget" name="cannyAnalysisWidget">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>331</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Canny parameters</string>
      </property>
      <widget class="QSlider" name="cannyBlurSlider">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Blur</string>
       </property>
      </widget>
      <widget class="QLabel" name="cannyBlurLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Low</string>
       </property>
      </widget>
      <widget class="QLabel" name="cannyLowLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QSlider" name="cannyLowSlider">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="cannyHighLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
      <widget class="QSlider" name="cannyHighSlider">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>High</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Canny Edge Detection</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="overlayOriginalCanny">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Overlay with original image</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Use channels above to select channel for Edge Detection</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="OtsuAnalysisWidget">
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Otsu Tresholding</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Use channels above to select channel for Otsu thresholding</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="overlayOriginalOtsu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Overlay with original image</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Otsu parameters</string>
      </property>
      <widget class="QSlider" name="otsuBlurSlider">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Blur</string>
       </property>
      </widget>
      <widget class="QLabel" name="otsuBlurLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1210</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionCanny_Edge_Detection"/>
    <addaction name="actionOtsu_thresholding"/>
    <addaction name="actionAn"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalysis"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCanny_Edge_Detection">
   <property name="text">
    <string>Canny Edge Detection</string>
   </property>
  </action>
  <action name="actionOtsu_thresholding">
   <property name="text">
    <string>Otsu thresholding</string>
   </property>
  </action>
  <action name="actionAn">
   <property name="text">
    <string>An</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
